zgrab_schema.py

```diff
@@ -202,7 +202,8 @@
     }),
     "session_ticket":SubRecord({
         "value":Binary(),
-        "length":Integer()
+        "length":Integer(),
```

Comment:
I think this is determined more by "does it vary wildly" between hosts. For
something like keys, it does. For other things, where it's always the same,
it doesn't matter. So, I would argue that we have it if it varies.

On Tue, Jan 5, 2016 at 6:29 PM David Adrian <notifications@github.com>
wrote:

> In zgrab_schema.py
> <https://github.com/zmap/zgrab/pull/106#discussion_r48922057>:
>
> > @@ -202,7 +202,8 @@
> >      }),
> >      "session_ticket":SubRecord({
> >          "value":Binary(),
> > -        "length":Integer()
> > +        "length":Integer(),
>
> We usually only store length (in bits) for primes and keys. We've removed
> it in other places, such as the ZMap JSON output.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zmap/zgrab/pull/106/files#r48922057>.
>